tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -37,7 +37,7 @@ var (
 	// The CI-operator uses AWS region `us-east-1` which has the corresponding image ID: ami-0b8d82dea356226d3 for
 	// Microsoft Windows Server 2019 Base with Containers.
 	imageID      = "ami-0b8d82dea356226d3"
-	instanceType = "m4.large"
+	instanceType = "m5a.large"
```

Comment:
We might want to preserve the knowledge somewhere, why we are using AMD based instance on AWS